src/coreclr/inc/corinfo.h

```diff
@@ -2221,6 +2221,7 @@ class ICorStaticInfo
     // should be looked up at runtime.
     virtual void expandRawHandleIntrinsic(
         CORINFO_RESOLVED_TOKEN *        pResolvedToken,
+        CORINFO_METHOD_HANDLE           containingFtn,
```

Comment:
Should this be called `callerHandle`? It is what we call it in several existing methods.

Reply:
Addressed